DOC-1839/partner-dashboard-display-onboarding-and-kyc-notification (#653)
swan-io/docs: docs/topics/onboarding/account-holders/index.mdx

**Trigger**: Manual request by a Swan KYC analyst after the user completes identification but before the account holder verification status changes to `Verified`.

> **Configuration**: Swan can set up this notification for you based on your communication preferences. The available options include sending notifications directly to your customer, to you as the partner, copying both parties, or using the API to manage communications yourself. You can view the current configuration in your **Dashboard** under **Settings** > **Notifications**; however, it cannot be edited there. To make any updates, please reach out to us.

Author: ```suggestion
**Configuration**: Swan can set up this notification for you based on your communication preferences. The available options include sending notifications directly to your customer, to you as the partner, copying both parties, or using the API to manage communications yourself. You can view (but can't edit) the current configuration in your **Dashboard** under **Settings** > **Notifications**. To make any updates, please reach out to us.
```

> **Configuration**: Swan can configure this notification on your behalf based on your preferred communication flow. Contact us to discuss your preference. The available options include sending notifications directly to the end user, to you as the partner, copying both parties, or using the API to manage communications yourself. You can view its configuration in your **Dashboard** under **Settings** > **Notifications**, but it can't be edited there. To update it, contact Swan.

Reviewer: ```suggestion
**Configuration**: Swan can set up this notification for you based on your communication preferences. The available options include sending notifications directly to your customer, to you as the partner, copying both parties, or using the API to manage communications yourself. You can view the current configuration in your **Dashboard** under **Settings** > **Notifications**; however, it cannot be edited there. To make any updates, please reach out to us.
```
Tidying this up for conciseness.

**Trigger**: Manual request by a Swan KYC analyst when additional supporting documents are needed for an onboarding-related document collection.

> **Configuration**: Swan can configure this notification on your behalf. Contact your Account Manager to discuss your preferences. The available communication options are the same as for the first transfer notification. You can view its configuration in your **Dashboard** under **Settings** > **Notifications**, but it can't be edited there. To update it, contact Swan.

Reviewer: Same update as above - remove the line about account manager - we already say "contact swan" at the end.

> **Configuration**: Swan can configure this notification on your behalf. Contact your Account Manager to discuss your preferences. The available communication options are the same as for the first transfer notification. You can view its configuration in your **Dashboard** under **Settings** > **Notifications**, but it can't be edited there. To update it, contact Swan.

Author: ```suggestion
**Configuration**: Swan can configure this notification on your behalf. The available communication options are the same as for the first transfer notification. You can view its configuration in your **Dashboard** under **Settings** > **Notifications**, but it can't be edited there. To update it, contact Swan.
```
